libs/sdk-react-native/breez_sdk.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "11.0" }
+  s.platforms    = { :ios => "14.0" }
```

Comment:
This can be lower back to 11 also

Reply:
Lowered b60d13ad9375f7e451bed5290a66c28ff2f29014